pkg/storage/metrics.go

```diff
@@ -0,0 +1,110 @@
+/*
+ * Nuts crypto
+ * Copyright (C) 2020. Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package storage
+
+import (
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/nuts-foundation/nuts-crypto/log"
+	errors2 "github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// in minutes
+const checkInterval = 15
```

Comment:
you could use `time.Duration` here (though not sure if you can still make it a const, then)

Reply:
then it would become a var